app/cypress/integration/educational-materials.spec.js

```diff
@@ -0,0 +1,13 @@
+describe('Educational Materials tab', () => {
+  it('Visits the About page and checks for existence of links', () => {
+    cy.visit('/about');
+    const linkText =
+      'https://www.epa.gov/waterdata/hows-my-waterway-middle-school-lesson-plan';
+    cy.findByText(linkText).should('not.be.visible');
+
+    const educationTab = 'Educational Materials';
+    cy.findByText(educationTab).click();
+
+    cy.findByText(linkText).should('be.visible');
+  });
+});
```

Comment:
Can you move this test over to the `feature/40_educational-materials` branch associated with PR #605?

Reply:
@cschwinderg I just saw these requests, I'll be sure to make those changes first thing tomorrow.